qiskit/circuit/qpy_serialization.py

```diff
@@ -213,6 +273,8 @@
 ``qr`` would have ``standalone`` set to ``False``.
 
 
+.. custom_definition:
```

Comment:
missing `_` on the front?
```suggestion
.. _custom_definition:
```